OPHD/Things/Structures/Structure.h

```diff
@@ -134,6 +137,12 @@ class Structure: public Thing
 	void update() override;
 	virtual void think() {}
 
+	/**
+	* Pass limited structure specific details for drawing. Use a custom UI window if needed.
+	*/
+	#pragma warning(suppress: 4100) // Unreferenced formal parameter
+	virtual void drawInspectorView(NAS2D::Renderer& renderer, const NAS2D::Rectangle<float>& windowRect) {}
```

Comment:
Another way to avoid the warning is to comment out the parameter name:
```cpp
virtual void drawInspectorView(NAS2D::Renderer& /*renderer*/, const NAS2D::Rectangle<float>& /*windowRect*/) {}
```

Parameter names are of course optional in contexts where parameters aren't used.

----

Another possibility might be the attribute [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused), though I've never experimented with it.

Reply:
I tried maybe_unused and it seemed to work well on MSVC. I wanted to leave the argument names in place for context on what the NAS2D::Rectangle referenced.

Thanks for doing the research on how to resolve the warning/error.